pymatgen/io/vasp/MatPESStaticSet.yaml

```diff
@@ -0,0 +1,160 @@
+# Default VASP settings for static calculations in the MatPES Project
+# using the Regularized-Restored Strongly Constrained and Appropriately
+# Normed functional (PBE, PBE+U or r2SCAN).
+INCAR:
+  ALGO: ALL
```

Comment:
Default ALGO should be Normal for PBE. R2SCAN uses All. But our input set is written with PBE as a default.